public-api/db-cdr/generate-cdr/run-measurement-queries.sh

```diff
@@ -44,27 +44,610 @@ then
   exit 1
 fi
 
+#Get the list of tables in the dataset
+tables=$(bq --project=$BQ_PROJECT --dataset=$BQ_DATASET ls)
+
 # Next Populate achilles_results
 echo "Running measurement queries..."
 
+# Run these queries in prod to generate counts for ppi concepts from the full measurement table (and not just the ehr one used to generate ehr counts for rest of the concepts)
+# In case of test data, the views has all the data, so we do not need to run these.
+# If mapping tables are present, the views contain ehr specific data, so we need to run these extra queries.
+if [[ "$tables" == *"_mapping_"* ]]; then
+    # 3000 Measurements that have numeric values - Number of persons with at least one measurement occurrence by measurement_concept_id, bin size of the measurement value for 10 bins, maximum and minimum from measurement value. Added value for measurement rows
+    bq --quiet --project=$BQ_PROJECT query --nouse_legacy_sql \
```

Comment:
Where did all of this new code come from? Was this a copy paste from somewhere?

Reply:
this queries are exactly run below for rest of the ehr measurements considering only the ehr record rows in v_ehr_measurement. I wanted these to run on the full measurement table to calculate the counts of just the physical measurements. (which do not have counts if these queries are not run)